### ISSUE TITLE
CRD name changes in cleanup for RHODS install doc

### ISSUE THREAD

**Issue by jbusche** [red-hat-data-services/distributed-workloads#11]
Small changes needed for crd cleanup in the quickstart docs...
WAS:
schedulingspecs.mcad.ibm.com appwrappers.mcad.ibm.com quotasubtrees.ibm.com

but now is:
quotasubtrees.quota.codeflare.dev
appwrappers.workload.codeflare.dev
schedulingspecs.workload.codeflare.dev

**Comment by jbusche** [comment]
PR created here:  https://github.com/red-hat-data-services/distributed-workloads/pull/28

**Comment by jbusche** [comment]
Shoot, we got a new high vulnerability, patching that one too.  
```
High,Has fix,High severity,Recent vulnerability  CVE-2023-45142 https://nvd.nist.gov/vuln/detail/CVE-2023-45142 Y   fixed in 0.44.0
```
